bundles/org.openhab.binding.bluetooth.airthings/src/main/java/org/openhab/binding/bluetooth/airthings/internal/AirthingsWaveGen1Handler.java

```diff
@@ -0,0 +1,134 @@
+/**
+ * Copyright (c) 2010-2021 Contributors to the openHAB project
+ *
+ * See the NOTICE file(s) distributed with this work for additional
+ * information.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ */
+package org.openhab.binding.bluetooth.airthings.internal;
+
+import static org.openhab.binding.bluetooth.airthings.internal.AirthingsBindingConstants.*;
+
+import java.util.UUID;
+
+import org.eclipse.jdt.annotation.NonNullByDefault;
+import org.openhab.core.library.types.QuantityType;
+import org.openhab.core.library.unit.SIUnits;
+import org.openhab.core.library.unit.Units;
+import org.openhab.core.thing.Thing;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+/**
+ * The {@link AirthingsWaveGen1Handler} is responsible for handling commands, which are
+ * sent to one of the channels.
+ *
+ * @author Davy Wong - Added Airthings Wave Gen 1 support
+ */
+@NonNullByDefault
+public class AirthingsWaveGen1Handler extends AbstractAirthingsHandler {
+
+    private static final String HUMIDITY_UUID = "00002a6f-0000-1000-8000-00805f9b34fb"; // 0x2A6F
+    private static final String TEMPERATURE_UUID = "00002a6e-0000-1000-8000-00805f9b34fb"; // 0x2A6E
+    private static final String RADON_STA_UUID = "b42e01aa-ade7-11e4-89d3-123b93f75cba";
+    private static final String RADON_LTA_UUID = "b42e0a4c-ade7-11e4-89d3-123b93f75cba";
+
+    private int intResult;
+    private double dblResult;
+    private volatile ReadSensor readSensor = ReadSensor.RADON_STA;
+
+    private enum ReadSensor {
+        TEMPERATURE,
+        HUMIDITY,
+        RADON_STA,
+        RADON_LTA,
+    }
+
+    public AirthingsWaveGen1Handler(Thing thing) {
+        super(thing);
+    }
+
+    private final Logger logger = LoggerFactory.getLogger(AirthingsWaveGen1Handler.class);
+
+    @Override
+    protected void updateChannels(int[] is) {
+        int[] rawdata;
+        rawdata = is;
+        if (rawdata.length == 2) {
+            switch (readSensor) {
+                case TEMPERATURE:
+                    dblResult = intFromBytes(rawdata[0], rawdata[1]) / 100D;
+                    logger.info("Parsed data 1: {}", String.format("[temperature=%.1f °C]", dblResult));
```

Comment:
Please only use debug level for all those logs (here and below).

Reply:
I changed all logging to debug level only, as requested.